src/components/Movies/MoviesList/MovieItem/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './style.css';
+
+class MovieItem extends React.Component {
+    render() {
+        return (
+            <div className='movieItem' style={this.props.style}>
+                <img className='movieItem_poster' src={this.props.movie.poster_path} />
```

Comment:
Not necessary to change but may be it would be better to use properties de-structuring fo movie properties as an example.